hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/dao/NodeLabelsInfo.java

```diff
@@ -33,7 +33,7 @@ public class NodeLabelsInfo {
 
   @XmlElement(name = "nodeLabelInfo")
   private ArrayList<NodeLabelInfo> nodeLabelsInfo =
```

Comment:
Single line?

Reply:
Thanks for your help reviewing the code, I will fix it.